### ISSUE TITLE
Incorrectly Identifies `Julia(.jl)` files as `Python` file

### ISSUE THREAD

**Issue by chayandatta** [google/magika#49]
![image](https://github.com/google/magika/assets/32599474/bd251c66-9afc-4fe5-af34-fcafab0d3837)

Unzip the attached `.jl` file to try yourself. (I had to compress the file, else GitHub wouldn't let me upload it.)

[AutoJuliaDemo (4).jl.zip](https://github.com/google/magika/files/14310983/AutoJuliaDemo.4.jl.zip)

**Comment by niutech** [comment]
Try any example from [this page](https://juliabyexample.helpmanual.io/). Magika probably wasn't trained on Julia code.

**Comment by reyammer** [comment]
No, it was not trained on julia. Noting this down for v2, we'll see if we can include it. Thanks!

**Comment by invernizzi** [comment]
fixed in model v3.3

<img width="1193" height="360" alt="Image" src="https://github.com/user-attachments/assets/705b277c-2b37-4956-9014-76483756bfb2" />
